### ISSUE TITLE
UZG stream won't play on CoreELEC after Widevine update

### ISSUE THREAD

**Issue by Markman-B** [Opvolger/plugin.video.uzg#41]
Yesterday, I received a popup message to update Widevine. The update went fine. But after the update I'm unable to watch UZG streams. Anyone else experiencing the same issue.

Please tell me what info you need, if you're willing to investigate the issue. 
Regards,
Mark

Snippet logfile:
T:2925904768 ERROR: AddOnLog: InputStream Adaptive: Unable to load widevine shared library (/storage/.kodi/cdm/libwidevinecdm.so) 2020-10-03 14:22:34.087 

T:2925904768 ERROR: AddOnLog: InputStream Adaptive: OpenDRMSystem failed 2020-10-03 14:22:34.088 

T:2925904768 ERROR: CVideoPlayer::OpenInputStream - error opening [plugin://plugin.video.uzg/?action=play&whatson_id=POW_04506082] 2020-10-03 14:22:34.088 

T:2925904768 NOTICE: CVideoPlayer::OnExit() 2020-10-03 14:22:34.102 

T:4064108560 NOTICE: CVideoPlayer::CloseFile() 2020-10-03 14:22:34.108 

T:4064108560 NOTICE: VideoPlayer: waiting for threads to exit 2020-10-03 14:22:34.108 

T:4064108560 NOTICE: VideoPlayer: finished waiting 2020-10-03 14:22:34.119 

EDIT: Restoring Widevine to version 13020.87.0 from the Inputstream Helper addon, seems to fix the issue for me.

**Comment by ErikBrown2** [comment]
I had an issue till today that most NPOstart programs did not work anymore on Retrospect. I live outside Europe and it appears that NPO has added some drm authentication for most programs for viewers from outside Europe. The issue was seen with Retrospect and Kodi on my Raspberry Pi but not with Kodi and Retrospect on my PC. It turned out that there was a bug with the Inputstream Helper add-on for ARM CPUs, so also for the Raspberry Pi. Upgrading the Inputstream Helper add-on from version 0.5.0 to 0..5.1 solved the issue. See https://github.com/xbmc/repo-scripts/pull/1700 for this issue.

**Comment by Markman-B** [comment]
Thanks for the information. I've followed procedure you mentioned and have updated the Inputstream Helper addon. I received a message to update Widevine, as soon as I tried to play the NOS Journaal. After the update the NOS Journaal stream opened fine.

**Comment by bartmans** [comment]
I have the same issue now on a RPi4 running LibreEelec, Kodi 18.9. Updated all relevant plugins to latest versions (UZG, InputstreamHelper), but I have no luck watching shows like NOS Journaal or Kamp Koningsbrugge. Watching on an Windows 10 system had no such problems.

I attached a log with the relevant error. I also use Retrospect, this has the same issue as of today.

[kodi.log](https://github.com/Opvolger/plugin.video.uzg/files/5683804/kodi.log)
